mlflow/models/model.py

```diff
@@ -29,55 +30,124 @@
 _MLFLOW_VERSION_KEY = "mlflow_version"
 
 
-class ModelInfo(NamedTuple):
+class ModelInfo:
     """
     The metadata of a logged MLflow Model.
     """
 
-    #: Run relative path identifying the logged model.
-    artifact_path: str
+    def __init__(
+        self,
+        artifact_path: str = None,
+        flavors: Dict[str, Any] = None,
+        model_uri: str = None,
+        model_uuid: str = None,
+        run_id: str = None,
+        saved_input_example_info: Optional[Dict[str, Any]] = None,
+        signature_dict: Optional[Dict[str, Any]] = None,
+        signature=None,  # Optional[ModelSignature]
+        utc_time_created: str = None,
+        mlflow_version: str = None,
+    ):
+        """
+        :param artifact_path: Run relative path identifying the logged model.
+        :param flavors: A dictionary mapping the flavor name to how to serve
+                        the model as that flavor. For example:
```

Comment:
The `ModelInfo` doc looks like this:

<img width="1095" alt="image" src="https://user-images.githubusercontent.com/17039389/186329606-58697be8-e455-4b84-8e19-12458281f12a.png">


Can we fix it?

Reply:
Ah I see the properties are not rendered, will fix.